assets/js/admin/orders/order-overview/models/state.js

```diff
@@ -161,9 +187,16 @@ export const State = Backbone.Model.extend(
 						return amount += +adjustment.get( 'subtotal' );
 					}
 				},
-				this.getSubtotal()
+				this.getSubtotal( includeTaxInSubtotal )
 			);
 
+			if ( true === this.hasInclusiveTax() ) {
+				// Fees always have tax added.
```

Comment:
I'm not sure if this is true anymore, since it was changed to a toggle as described in this [comment](https://github.com/easydigitaldownloads/easy-digital-downloads/pull/8581#issuecomment-827792453). Same for the comment in order-adjustment.js.

Reply:
@LisaCee I think the comment should really be:

```
// Fees always have tax added exclusively
```

You can choose to add tax or not, but if you do add tax, it will be added to the entered value (exclusive tax).